src/App.svelte

```diff
@@ -24,6 +24,7 @@
   export let url = "/";
   const tabRepository = new TabRepository();
   let flag: boolean = true;
+  let isActiveInternet: boolean = true;
```

Comment:
no need to specify type here.

let isActiveInternet = true; should do

Reply:
followed the trend :-P
will update in next PR

Reply:
okk